openmdao/core/system.py

```diff
@@ -3613,6 +3630,10 @@ def list_outputs(self,
             Set to None to suppress.
         values : bool, optional
             This argument has been deprecated and will be removed in 4.0.
+        print_min : bool
+            When true, if the output value is an array, print its smallest value.
+        print_max : bool
+            When true, if the output value is an array, print its largest value.
```

Comment:
I would add a little example to the notebook `features/debugging/listing_variable`, probably right under the section that shows how to print array variables.  Just an example that shows the min and max of the array vars.